assets/js/ajax-filters.js

```diff
@@ -27,21 +27,6 @@ jQuery( document ).ready( function ( $ ) {
 			xhr[index].abort();
 		}
 
-		if ( ! append ) {
-			$( results ).addClass( 'loading' );
-			$( 'li.job_listing, li.no_job_listings_found', results ).css( 'visibility', 'hidden' );
-
-			// Not appending. If page > 1, we should show a load previous button so the user can get to earlier-page listings if needed
```

Comment:
These lines are responsible for outputting and handling the text that was requested to be removed from the page (also the set of matched elements is not well defined in the .wrap() jQuery element, so it is not functional).